### PR TITLE
Update README.md | Typos fixed

### PR DESCRIPTION
**Pull request by imsaptarshi**
## Summary

Some typos like, stricly(strictly), Licence(License), Initialise(Initialize)... 
we're fixed! Thank me later.

## Checklist

- [ ] If endpoints were changed then they have been documented and tested.
    - [ ] I have updated the docmentation to reflect the changes.
    - [ ] I have updated the tests to support the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new endpoint or parameter).
- [ ] This PR is a breaking change (e.g. endpoint or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ And fill in the variables with the values below:
 
 ### Running
 
-Run the API and initialise the database:
```
Comment (takos22) [inline README.md]: We are using British English so `initialise` is correct.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [![Status](https://img.shields.io/website?url=https%3A%2F%2Fapi.dev.twtcodejam.net)](https://api.dev.twtcodejam.net) <!-- TODO: Switch to main API link. -->
 [![GitHub Issues](https://img.shields.io/github/issues/Tech-With-Tim/API.svg)](https://github.com/Tech-With-Tim/API/issues)
 [![GitHub Pull Requests](https://img.shields.io/github/issues-pr/Tech-With-Tim/API.svg)](https://github.com/Tech-With-Tim/API/pulls)
-[![Licence](https://img.shields.io/badge/licence-MIT-blue.svg)](/LICENCE)
```
Comment (takos22) [inline README.md]: We are using British English so `licence` is correct.

File: README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ API for the Tech With Tim website using [Quart](https://pgjones.gitlab.io/quart/
 - [✅ Linting](#-linting)
 - [🚨 Tests](#-tests)
 - [📚 Docs](/docs/README.md)
-- [📜 Licence](/LICENCE)
```
Comment (takos22) [inline README.md]: We are using British English so `licence` is correct.

**Review by takos22 — CHANGES_REQUESTED** [review]
We are using British English so `licence` and `initialise` are correct.

File: README.md (hunk 3 above)
Comment (imsaptarshi) [reply README.md]: Got it fixed

**Review by takos22 — APPROVED** [review]
Ok thanks

**Pull request merged**